packages/office-ui-fabric-react/src/components/Button/ButtonCore/ButtonCoreVars.scss

```diff
@@ -1,3 +1,16 @@
 $button-core-default-height: 32px;
 $button-core-default-minWidth: 80px;
 $button-core-default-padding: 0 16px;
+
+@mixin core-button-disabled {
```

Comment:
Good point, it's not likely to be imported into someone else's app and there's a bunch of other places where we're not prefixing variables/mixins. Let's leave it.